src/overview.md

```diff
@@ -250,7 +250,7 @@ Moreover, the compiler wasn't originally built to use a query system; the query
 system has been retrofitted into the compiler, so parts of it are not query-fied
 yet. Also, LLVM isn't our code, so that isn't querified either. The plan is to
 eventually query-fy all of the steps listed in the previous section, but as of
-April 2020 <!-- date: 2020-04 -->, only the steps between HIR and LLVM-IR are
+February 2021 <!-- date: 2021-02 -->, only the steps between HIR and LLVM IR are
 query-fied. That is, lexing and parsing are done all at once for the whole
 program.
```

Comment:
I *think* this is still true, but it would be good to check.

Reply:
`rustc_resolve` operates on the AST though, so I think this is actually correct as-is.

Reply:
Right, I meant that "lexing and parsing" is an incomplete list, it should also include name resolution.

Reply:
Ah, I see 👍

Reply:
Does this seem right to you?

```diff
diff --git a/src/overview.md b/src/overview.md
index c96321f..80968a6 100644
--- a/src/overview.md
+++ b/src/overview.md
@@ -251,8 +251,9 @@ system has been retrofitted into the compiler, so parts of it are not query-fied
 yet. Also, LLVM isn't our code, so that isn't querified either. The plan is to
 eventually query-fy all of the steps listed in the previous section, but as of
 February 2021 <!-- date: 2021-02 -->, only the steps between HIR and LLVM IR are
-query-fied. That is, lexing and parsing are done all at once for the whole
-program.
+query-fied. That is, lexing, parsing, name resolution (except for method call
+resolution, which happens in type-checking), and macro expansion (the last two
+happen as part of parsing) are done all at once for the whole program.
 
 One other thing to mention here is the all-important "typing context",
 [`TyCtxt`], which is a giant struct that is at the center of all things.
```

I know that at least some parts of macro expansion happen interleaved with parsing (plus I think it operates on the AST), so I would assume it's not query-driven.

Reply:
Hmm, this is more detailed but maybe less clear. I would just say "name resolution and macro expansion".